metaseq/tasks/streaming_finetune_language_modeling.py

```diff
@@ -106,7 +125,7 @@ def load_dataset(self, split: str, epoch=1, combine=False, **kwargs):
             block_size=self.args.tokens_per_sample + 1,
             break_mode=self.args.sample_break_mode
             if split == "train"
-            else "eos_pad_8",  # eos mode for val/test to get accuracies
+            else self.args.valid_sample_break_mode,  # use diferent mode for val/test
```

Comment:
What are the different modes for valid and test?

Reply:
outdated. No using this anymore